src/sqlittaaja/style.css

```diff
@@ -40,3 +40,21 @@ th {
 tbody tr:nth-child(even) {
   background-color: #f0f0f0;
 }
+
+/* Styles for difflib.HtmlDiff().make_table() output from diff_html.py */
```

Comment:
The styling doesn't seem to match perfectly. Maybe add some rounding or try to use the existing styles?

Reply:
Could you maybe also use Prettier to style the CSS?